translations/base-kor.yaml

```diff
@@ -111,136 +104,149 @@ dialogs:
-        text: 저장 파일을 삭제하지 못했습니다.
+        text:
+            세이브 파일을 삭제하지 못했습니다.
     restartRequired:
-        title: 다시 시작 필요
-        text: 설정을 적용하려면 게임을 다시 시작해야 됩니다.
+        title: 게임 재시작 필요
+        text:
+            설정을 적용하려면 게임을 재시작해야 합니다.
     editKeybinding:
-        title: 키바인딩 바꾸기
+        title: 조작법 바꾸기
         desc: 당신이 원하는 키나 마우스 버튼을 눌러서 바꾸거나 ESC를 눌러 취소하세요.
     resetKeybindingsConfirmation:
-        title: 키바인딩 재설정
-        desc: 이것은 모든 키바인딩을 기본값으로 초기화합니다.
+        title: 조작법 초기화
+        desc: 설정된 모든 조작법이 기본값으로 초기화됩니다. 초기화 전에 확인해 주세요.
     keybindingsResetOk:
-        title: 키바인딩 재설정 완료
-        desc: 모든 키바인딩이 기본값으로 재설정 되었습니다!
+        title: 조작법 초기화 완료
+        desc: 모든 조작법이 기본값으로 초기화 되었습니다!
     featureRestriction:
         title: 체험판 버전
-        desc: 체험판 버전에는 없는 콘텐츠(<feature>)로 시도했습니다. 유료 버전을 구입해서 모든 콘텐츠를 사용해보세요!
+        desc: 체험판 버전에는 없는 콘텐츠 (<feature>)로 시도했습니다. 정식 버전을 구입해서 모든 콘텐츠를 사용해보세요!
     oneSavegameLimit:
-        title: 저장파일 개수 제한
-        desc: 체험판 버전에서는 저장 파일을 한 번에 한 개만 사용할 수 있습니다. 이미 있는 저장 파일을 지우거나 유료 버전을 구입 해주새요.
+        title: 세이브 파일 개수 제한
+        desc: 체험판 버전에서는 세이브 파일을 한 번에 한 개만 사용할 수 있습니다. 이미 있는 세이브 파일을 지우거나 정식 버전을 구입 해주새요.
     updateSummary:
-        title: 신규 버전!
-        desc: 지난번 플레이 이후 변경사항은 다음과 같습니다.
+        title: 새로운 업데이트!
+        desc:
+            지난번 플레이 이후 변경 사항은 다음과 같습니다.
     upgradesIntroduction:
         title: 업그레이드 하기
         desc:
-            여러분이 만든 모든 도형은 업그레이드에 사용 될 수 있습니다! - <strong> 만들어 놓은 공장을 허물지 마세요!</strong>
+            <strong>기존의 공장을 허물지 마세요!</strong> 여러분이 그동안 만들어 수집한 모든 도형은 업그레이드에 사용됩니다.
             업그레이드 버튼은 화면의 오른쪽 위에 있습니다.
     massDeleteConfirm:
         title: 삭제 확인
-        desc: 당신은 많은 건물을 삭제하려고 하고있습니다! (정확히는 <count>개) 삭제하시겠습니까?
+        desc:
+            굉장히 많은 수 (정확히는 <count>개)의 건물을 삭제하려 합니다! 이 작업을 수행하시겠습니까?
     massCutConfirm:
         title: 자르기 확인
-        desc: 당신은 많은 건물을 자르려고 하고있습니다! (정확히는 <count>개) 자르시겠습니까?
+        desc:
+            굉장히 많은 수 (정확히는 <count>개)의 건물을 자르려고 합니다! 이 작업을 수행하시겠습니까?
     blueprintsNotUnlocked:
         title: 아직 사용 불가
-        desc: 복사 기능은 아직 열리지 않았습니다! 레벨을 올려서 잠금을 해제하세요.
+        desc:
+            12 레벨부터 청사진 기능이 해금됩니다!
     keybindingsIntroduction:
-        title: 유용한 키바인딩
-        desc: "큰 공장을 지을 때 유용한 키바인딩이 많습니다! 아래를 확인하세요.<strong>나머지 키바인딩도
-            확인해보세요!</strong>!<br><br> <code class='keybinding'>CTRL</code> +
-            Drag: 영역을 선택해서 복사/삭제하세요.<br> <code class='keybinding'>SHIFT</code>:
-            한꺼번에 여러 개의 건물을 배치하세요.<br> <code class='keybinding'>ALT</code>: 설치된
-            컨베이어 벨트의 방향을 바꾸세요.<br>"
+        title: 유용한 조작법
+        desc:
+            이 게임에는 거대한 공장을 수월하게 세우기 위한 많은 조작법이 있습니다.
+            아래는 그 대표적인 것이며, 자세한 조작법은 <strong>조작법 설정</strong>을 참고해주세요!<br><br>
+            <code class='keybinding'>CTRL</code> + 드래그: 영역을 선택합니다.<br>
+            <code class='keybinding'>SHIFT</code>: 누르는 동안 같은 건물을 여러개 배치할 수 있습니다.<br>
+            <code class='keybinding'>ALT</code>: 배치된 벨트의 방향을 반전시킵니다.<br>
```

Comment:
You probably need to add the `>-` after the `desc`, since yaml is seeing the parts like `<code class='keybinding'>CTRL</code> + 드래그:` as a yaml key because of the `:`, so it looks like this:

```
        desc: >-
            이 게임에는 거대한 공장을 수월하게 세우기 위한 많은 조작법이 있습니다.
            아래는 그 대표적인 것이며, 자세한 조작법은 <strong>조작법 설정</strong>을 참고해주세요!<br><br>
            <code class='keybinding'>CTRL</code> + 드래그: 영역을 선택합니다.<br>
            <code class='keybinding'>SHIFT</code>: 누르는 동안 같은 건물을 여러개 배치할 수 있습니다.<br>
            <code class='keybinding'>ALT</code>: 배치된 벨트의 방향을 반전시킵니다.<br>
```